salt/modules/quota.py

```diff
@@ -205,7 +205,7 @@ def on(device):
         salt '*' quota.on
     """
     cmd = "quotaon {0}".format(device)
-    __salt__["cmd.run"](cmd, python_shell=False)
+    __salt__["cmd.run"](cmd, python_shell=False, success_retcodes=[2])
```

Comment:
Only `2` is a valid success return code?
Because if `0` is also a success return code, you have to pass both.

Reply:
If that is the case, the docstring for cmd() needs to be fixed as it says to only list non-zero return codes.

Reply:
This is what is in the docstring:

`    :param list success_retcodes: This parameter will be allow a list of
        non-zero return codes that should be considered a success.  If the
        return code returned from the run matches any in the provided list,
        the return code will be overridden with zero.

      .. versionadded:: 2019.2.0
`
According to these docs, my commit is correct. I will take a look at the code and make sure this docstring isn't wrong.

Reply:
I have confirmed that my commit is correct. It is only necessary to list the non-zero returncodes that should be considered the same as zero. If the returncode is zero, it will still be returned as zero. The only thing that success_retcodes does, is -- just as it is described as a docstring -- morph any of the supplied non-zero returncodes into a zero.

Reply:
Guess I skimmed through that code too fast then.